### PR TITLE
host: Expose container PID

### PR DESCRIPTION
**Pull request by josephglanville**
This exposes the container PID via the host API.

Necessary for providing PID -> container mapping in tools such as Scope that need to join on the host namespace PID.

### CODE REVIEW

**Comment by josephglanville** [comment]
@lmars can you think of any edge cases that need to be handled here?
I have it setting the PID back to 0 in the state when transititioning to non-running states but I couldn't think of anything else off the top of my head.

File: host/types/types.go [hunk 1]
```diff
@@ -214,6 +214,7 @@ type ActiveJob struct {
 	Job        *Job      `json:"job,omitempty"`
 	HostID     string    `json:"host_id,omitempty"`
 	InternalIP string    `json:"internal_ip,omitempty"`
+	PID        int       `json:"pid,omitempty"`
```
Comment (lmars) [inline host/types/types.go]: What do you think about making this a pointer like `ExitStatus`? Then it is set iff the job is running.

Comment (josephglanville) [inline host/types/types.go]: I like that, `nil` is much more appropriate than 0 for a non-running job.

**Comment by josephglanville** [comment]
@lmars updated to use `*int` for PID.

**Pull request merged**